client/canal_connector.go

```diff
@@ -16,7 +16,8 @@
 
 package client
 
-import pb "github.com/CanalClient/canal-go/protocol"
+//import pb "github.com/CanalClient/canal-go/protocol"
```

Comment:
i think that you should  remove it(`//import pb "github.com/CanalClient/canal-go/protocol`).

Reply:
Yes，I removed it (`//import pb "github.com/CanalClient/canal-go/protocol`)